includes/class-wc-payments-tasks.php

```diff
@@ -31,9 +33,10 @@ public static function init() {
 	 */
 	public static function add_task_disputes_need_response() {
 		$account_service = WC_Payments::get_account_service();
-		if ( ! $account_service || ! $account_service->is_stripe_account_valid() ) {
+		if ( ! $account_service || ! $account_service->is_stripe_account_valid() || $account_service->is_account_under_review() || $account_service->is_account_rejected() ) {
 			return;
 		}
+		include_once WCPAY_ABSPATH . 'includes/admin/tasks/class-wc-payments-task-disputes.php';
```

Comment:
Thanks for clarifying the details of the problem and how you've optimised it @brucealdridge ! 🚀 

I think it's worth adding a comment in this function about why the include and task is deferred, i.e. leave breadcrumbs in the code about the performance problem.